foundry.toml

```diff
@@ -10,9 +10,10 @@ remappings = [
   'contracts/=contracts/',
 ]
 optimizer_runs = 15000
-fuzz_runs = 1000
-fuzz_max_global_rejects = 1000000
-fuzz_max_local_rejects = 1000000
```

Comment:
If I remember correctly the `fuzz_max_local_rejects` was needed for some other tests

Reply:
do you have issues with this flag on?

Reply:
i typed `forge config` and checked docs. 
i guess `fuzz_max_local_rejects` config was removed.
my forge version is 0.2.0. 

https://book.getfoundry.sh/reference/config/testing?highlight=max%20fuzz#fuzz

```
forge config 
warning: Key `fuzz.max_global_rejects` is being deprecated in favor of `fuzz.max_test_rejects`. It will be removed in future versions.
...
[fuzz]
runs = 1000
max_test_rejects = 1000000
max_global_rejects = 1000000
dictionary_weight = 40
include_storage = true
include_push_bytes = true
...
```

Reply:
ok